force-app/main/default/classes/ListOfSObjectsInOrg.cls

```diff
@@ -0,0 +1,9 @@
+public class ListOfSObjectsInOrg {
```

Comment:
please explain this class

Reply:
in this class there is a mathod which is returning the list of all the standard objects in the org